sqlglot/tokens.py

```diff
@@ -50,6 +50,7 @@ class TokenType(AutoName):
     PARAMETER = auto()
     SESSION_PARAMETER = auto()
     NATIONAL = auto()
+    CONTAINS = auto()
```

Comment:
should this be array contains?

Reply:
Sure

Reply:
maybe atlt?

Reply:
What is atlt?

Reply:
less than at

so ltat, what do we do for the other json ops?

Reply:
> why is there only one token instead of two?

It made sense to me because it's roughly the same relation, just with the operands reversed.

Reply:
isn’t that the same as < and > ?

Reply:
Done (discussed in slack)